Sources/LiveKit/protocols/TrackDelegate.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+#if !os(macOS)
+import UIKit
+#endif
+
+// Currenlty used for internal purposes
```

Comment:
nit: typo :)

Reply:
Oh yes oniichan ! I almost forgot, we should have a Policy. For Flutter also 🙂 I added to Linear.